remotespark/pysparkkernel/kernel.js

```diff
@@ -0,0 +1 @@
+IPython.CodeCell.config_defaults.highlight_modes['magic_text/x-sql'] = {'reg':[/^%%sql/, /^%%hive/]};
```

Comment:
You mean syntax highlight HQL? We just use the SQL lexer.
Sorry if my answer isn't clear, but I'm not sure I understood. Use it for what?